doc/crm.8.adoc

```diff
@@ -2104,6 +2104,71 @@ Example:
 utilization xen1 set memory 4096
 ...............
 
+[[cmdhelp.sbd,SBD management]]
+=== `sbd` - SBD management
+
+This level is for managing the SBD (STONITH Block Device) daemon.
+
+[[cmdhelp.sbd.configure,configure SBD]]
+==== `configure`
+
+Configure the SBD daemon for both disk-based and disk-less mode.
+
+Main functionailities include:
+- Show configured disk metadata
+- Show contents of /etc/sysconfig/sbd
+- Show SBD related cluster properties
+- Newly setup SBD configuration on a running cluster
+- Update the existing parameters
+- Add more devices to the existing disk-based SBD configuration
+
+For more details on SBD and related parameters, please see man sbd(8).
+
+Usage:
+...............
+# For disk-based SBD
+crm sbd configure show [disk_metadata|sysconfig|property]
+crm sbd configure [device=<dev>]... [watchdog-device=<dev>] [watchdog-timeout=<integer>] [allocate-timeout=<integer>] [loop-timeout=<integer>] [msgwait-timeout=<integer>]
+
+# For disk-less SBD
+crm sbd configure show [sysconfig|property]
+crm sbd configure device="" [watchdog-device=<dev>] [watchdog-timeout=<integer>]
+...............
+
+example:
+...............
+configure show
+configure show disk_metadata
+configure show sysconfig
+configure show property
+configure device="/dev/sdb1;/dev/sdb2"
+configure device=/dev/sdb1 device=/dev/sdb2
+configure device=/dev/sdb1 watchdog-timeout=30 msgwait-timeout=60
+configure device="" watchdog-timeout=30
+...............
+
+[[cmdhelp.sbd.status,show SBD status]]
+==== `status`
+
+Show the status of the SBD daemon.
```

Comment:
Buggy when the cluster is in the maintenance mode,

```
tws-corosync3-2:~ # crm_mon -rR1

Full List of Resources:
  * stonith-sbd (stonith:fence_sbd):     Stopped (maintenance)

tws-corosync3-2:~ # crm sbd status
fence_sbd status: 
(Empty result)
```

Reply:
> Buggy when the cluster is in the maintenance mode,
> 
> ```
> tws-corosync3-2:~ # crm_mon -rR1
> 
> Full List of Resources:
>   * stonith-sbd (stonith:fence_sbd):     Stopped (maintenance)
> 
> tws-corosync3-2:~ # crm sbd status
> fence_sbd status: 
> (Empty result)
> ```

That's because crm_resource direct info into stderr when resource NOT running
```
# crm_resource --locate --resource stonith-sbd 1> /dev/null
resource stonith-sbd is NOT running
```
I will try to catch both stdout and stderr here